src/App.js

```diff
@@ -1,26 +1,93 @@
-import React from 'react';
-import logo from './logo.svg';
+import React, {Component} from 'react';
 import './App.css';
+import 'bulma/css/bulma.css';
+import foods from './foods.json';
+import FoodList from './components/FoodList'
+import AddFood from './components/AddFood';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+
+
+export default class App extends Component {
+  
+  state={
+    foodList: foods,
+    filteredList: foods,
+    showForm: false,
+    showPage: false,
+  };
+
+  handleAdd = (event) => {
+    event.preventDefault();
+    const {foodList, filteredList} = this.state
+    let newName = event.target.foodName.value; 
+    let newCalories = event.target.foodCalories.value
+    let newImage = event.target.foodImage.value
+    
+    let newDish = {
+      name: newName,
+      calories: newCalories,
+      image: newImage,
+      quantity: 1
+
+    };
+    this.setState({
+      foodList: [newDish, ...foodList],
+      filteredList: [newDish, ...filteredList],
+      showForm: false,
+    })
+    }
+  
+handleFormShow = () => {
```

Comment:
this function should be indented one space to the side, since it is inside the class component.